phpstan.neon

```diff
@@ -0,0 +1,3 @@
+parameters:
```

Comment:
Better add here:
```
parameters:
    level: 0
    paths:
        - src
        - tests
```
and rename the file to `phpstan.neon.dist`

Reply:
What is the benefit of having a `.dist` file?

Reply:
Exactly the same as for `.php_cs.dist` or `phpunit.xml.dist` - to test new config locally, e.g. you can increase the level or try some exclude - we would have some - like in [`Tokens::insertAt`](https://github.com/phpstan/phpstan/issues/2106).

Reply:
What prevents one from editing the `phpstan.neon` file and eventually commit changes? I never saw a legit use case for `.php_cs.dist`. `phpunit.xml.dist` makes sense when one wants custom local options for e.g. output, but PHPStan does not have such option I think.

Reply:
It follows the pattern of "config files like these *with* `.dist` are to be committed while *without* are to be ignored." I find that a nice separation, and enough reason to do it. (So hereby also the request to add the without version to `.gitignore`)

Reply:
I'm feeling like we would follow this pattern for the sake of following this pattern. Why would one want to maintain a specific local file rather than editing the tracked one? What use case would need to have a local file that permanently differs from the tracked one?

Reply:
I agree with @julienfalque - I never had a need for `.dist` files.

Reply:
I agree with @julienfalque too